app/src/components/Scheduling/ContactReview.js

```diff
@@ -0,0 +1,150 @@
+import React, { useEffect, useState } from 'react';
+import Header from './Header';
+import {
+  TextField,
+  FormControl,
+  Button,
+} from '../../store/index';
+import { useStyles } from './ContactReviewStyles';
+
+const HARD_CODED_DATA = {
+  caseWorker: 'Tracy',
+  firstName: 'Nina',
+  lastName: 'Mosley',
+  email: 'nina99@gmail.com',
+  phoneNumber: '+1 (213) 555-5555'
+}
+
+export default function ContactReview() {
+  const classes = useStyles();
+  const [inputs, setInputs] = useState({
+    caseWorker: '',
+    firstName: '',
+    lastName: '',
+    email: '',
+    phoneNumber: '',
+  })
+
+  const [editMode, toggleEdit] = useState({
+    email: false,
+    phoneNumber: false,
+  });
+
+  useEffect(() => {
+    setInputs(HARD_CODED_DATA);
+  }, [])
+
+  const onChange = (e) => {
+    setInputs({...inputs, [e.target.name]: e.target.value})
```

Comment:
I like the pattern of `{...inputs, [e.target.name]: e.target.value}` but I'd prefer calling this function `handleChange`, since this is what the react docs use: https://reactjs.org/docs/forms.html